Content.Shared/Chemistry/Solution.cs

```diff
@@ -17,6 +18,7 @@ public class Solution : IExposeData, IEnumerable<Solution.ReagentQuantity>
         // Most objects on the station hold only 1 or 2 reagents
         [ViewVariables]
         private List<ReagentQuantity> _contents = new List<ReagentQuantity>(2);
+        public ReadOnlyCollection<ReagentQuantity> Contents => _contents.AsReadOnly();
```

Comment:
Prefer this, it's how we do it in the rest of the codebase.
```suggestion
        public IReadOnlyList<ReagentQuantity> Contents => _contents;
```